src/js/WebParts/ProjectStats/ProjectStatsChart/ProjectStatsChartSettings/index.tsx

```diff
@@ -30,7 +29,8 @@ export default class ProjectStatsChartSettings extends React.Component<ProjectSt
         return (
             <div className="ms-Grid-row" hidden={this.props.hidden}>
                 <div className="ms-Grid-col ms-sm12">
-                    <CommandBar items={this._getItems()} farItems={this._getFarItems()} />
+                    {this.props.renderCommandBar &&
```

Comment:
Her vil det være ryddigere å ha hidden={!this.props.renderCommandBar} på diven rundt.